Makefile

```diff
@@ -252,6 +297,26 @@ vmlinux: $(vmlinux)
 bbl: $(bbl)
 fit: $(fit)
 
+.PHONY: openocd
+openocd: $(openocd)
+	$(openocd) -f $(confdir)/u540-openocd.cfg
+
+
+terminfo := $(buildroot_initramfs_wrkdir)/host/share/terminfo
+
+$(terminfo): $(target_gdb)
+	mkdir -p $(terminfo)
+	# hack for problems with https://git.buildroot.org/buildroot/commit/?id=b35ad5d0b45e5288f4019aeaa06b87ef0f2ef016
+	$(buildroot_initramfs_wrkdir)/host/bin/tic \
+		$(buildroot_initramfs_wrkdir)/build/ncurses-6.1/misc/terminfo.src \
+		-o $(terminfo)
+
+.PHONY: gdb gdb-u-boot
+gdb: $(target_gdb) $(terminfo)
+
+gdb-u-boot: $(target_gdb) $(terminfo)
+	$(target_gdb) -ex "set remotetimeout 240" -ex "target extended-remote localhost:3333" u-boot/u-boot
+
```

Comment:
The new makefile targets should not work because they require buildroot to be built first, but have no build dependency on build root.

It looks like they work by accident because the $(initramfs).d include at the end forces buildroot to always be built first, even if you type "make clean", which is odd that make clean has to force a build first before it removes it.  And because the .d file is placed in the work dir that clean removes, typing make clean again forces another build only to remove it again.